### ISSUE TITLE
Vehicle Live Risk Prediction

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#473]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Vehicle Live Risk Prediction
:red_circle: **Aim** : Create a prediction model for vehicle live risk based on given dataset.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/punyamodi/vehicle-live-risk-prediction
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Avdhesh-Varshney** [comment]
@abhisheks008 i would like to work on this issue. Could you please assign it to me?

Full name : Avdhesh Varshney
GitHub Profile Link : https://github.com/Avdhesh-Varshney
Participant ID (If not, then put NA) :
Approach for this Project : 
- Analysing the dataset.
- EDA processing.
- Filtering the dataset or performing feature engineering so that which can easily fit on the model.
- Feature selection using tree based models, etc.
- Applying the different type of the models like, random forest, decision tree regression, xgboost, knn classifier, svm classifier, **deep neural networks**, etc.
- Summarizing all the algorithms at the end of the notebook.

What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) KWOC

**Comment by abhisheks008** [comment]
One issue at a time @Avdhesh-Varshney

**Comment by CoderOMaster** [comment]
Hi @abhisheks008 
Full name : Keshav Arora 
GitHub Profile Link : https://github.com/CoderOMaster 
Participant Role : JWOC 2024
Approach for this Project:
Will implement decision tree,random forest,knn classifier to find best algorithm.
Feature scaling, encoding (if required),ANN can be also be tried.
Regards

**Comment by Yuvika-14** [comment]
hi abhisekh 
Full name: Yuvika Singh
git hub profile link:https://github.com/Yuvika-14/Yuvikademo.git
approch: ensemble methods, gradient boosting, neural networks.
Take care of the missing data if there are any categorical values then will use one hot encoder or label encoder depending upon the issue.

**Comment by abhisheks008** [comment]
> hi abhisekh Full name: Yuvika Singh git hub profile link:https://github.com/Yuvika-14/Yuvikademo.git approch: ensemble methods, gradient boosting, neural networks. Take care of the missing data if there are any categorical values then will use one hot encoder or label encoder depending upon the issue.

This is part of JWOC event.

**Comment by CoderOMaster** [comment]
Hi @abhisheks008
Full name : Keshav Arora
GitHub Profile Link : https://github.com/CoderOMaster
Participant Role : JWOC 2024
Approach for this Project:
Will implement decision tree,random forest,knn classifier to find best algorithm.
Feature scaling, encoding (if required),ANN can be also be tried.
Regards

**Comment by abhisheks008** [comment]
Issue assigned to you @CoderOMaster
